lib/src/core/routing/routing.dart

```diff
@@ -8,22 +13,61 @@ enum AppRoutes {
   course,
 }
 
-routerDelegate({String? initialPath}) => BeamerDelegate(
-      initialPath: initialPath ?? '/',
-      //TODO -LOW/MEDIUM- remove animation just on web
-      transitionDelegate: const NoAnimationTransitionDelegate(),
-      locationBuilder: RoutesLocationBuilder(
-        routes: {
-          '/': (_, __, ___) => const HomePage(),
-          '/archive': (_, __, ___) => const ArchivePage(),
-          '/course/:path': (_, state, __) {
-            final path = state.pathParameters['path']!;
-            return CoursePage(path);
-          },
-          '/article/:path': ((_, state, __) {
-            final path = (state.pathParameters['path']!);
-            return ArticlePage(path: path);
-          }),
-        },
-      ),
-    );
+final routerDelegate = BeamerDelegate(
+  locationBuilder: RoutesLocationBuilder(
+    routes: {
+      // Return either Widgets or BeamPages if more customization is needed
+      '/:locale': (_, state, ___) {
+        final languageCode = (state.pathParameters['locale']!);
+        return LocaleWrapper(
+          languageCode: languageCode,
+          child: const HomePage(),
+        );
+      },
```

Comment:
In order to change locale with path, the pages needs to be wrapped with LocaleWrapper

Reply:
I want a class of beamer! hehe

Reply:
Me too!!

Reply:
hahahahah